src/core/client/admin/components/StoryInfoDrawer/StoryInfoDrawerQuery.tsx

```diff
@@ -33,6 +39,10 @@ const StoryInfoDrawerQuery: FunctionComponent<Props> = ({
           return <QueryError error={error} />;
         }
 
+        if (!props?.viewer) {
+          return <h1>TODO NO VIEWER?!</h1>;
```

Comment:
I think i can assume that there will be a viewer here since Admin is an authenticated view, so I'm going to just assert not null, but let me know if that seems problematic